changelog.md

```diff
@@ -154,6 +154,8 @@ provided by the operating system.
   arguments passed to it, so `initOptParser` has been changed to raise
   `ValueError` when the real command line is not available. `parseopt` was
   previously excluded from `prelude` for JS, as it could not be imported.
+- Added `os.isAdmin` to tell whether the caller's process is a member of the
```

Comment:
newline between unrelated entries, and this should probably go below
```
`Added optional `followSymlinks` argument to `setFilePermissions`.
```
to group changelog entries related to os together (assuming this stays in std/os, that is)